packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -7,6 +7,22 @@ import cx from 'classnames';
 import { useMergedRefs, useTheme } from '../utils';
 import '@itwin/itwinui-css/css/toggle-switch.css';
 
+type ToggleSwitchSizeProps =
```

Comment:
is this intermediate type necessary? could be inlined into `ToggleSwitchProps`

Reply:
I actually don't know. I was referring to this example in `Tabs.tsx` :
![image](https://user-images.githubusercontent.com/22462084/224098938-de865c5a-f31f-4001-bc22-1d5611f964d9.png)
And the way I understand it (correct me if I'm wrong) is that this intermediate typing would separate icon from different sizing only. But if we combine it into one type, then "small" size will be separated from all the other props too not just icon. Actually let me test that rn 🤔

Reply:
ohh never mind, it works the same with inline typing! updated